app/core/pokemon-state.ts

```diff
@@ -140,22 +156,6 @@ export default abstract class PokemonState {
         totalTakenDamage += takenDamage
       }
 
```

Comment:
then after the trueDamagePart computed, line 158:
```suggestion
if (attackType === AttackType.SPECIAL) {
        specialDamage = damage
      } else {
        physicalDamage = damage
      }
```